logger/types.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  types.swift
+//  Test
+//
+//  Created by Daniel Brain on 10/12/20.
+//
+
+import Foundation
+
+typealias Payload = [String: String]
+typealias Headers = [String: String]
+
+enum LogLevel: String {
+    case debug, info, warn, error
```

Comment:
I would suggest keeping these in alphabetical order. I doubt this enum will expand much further, but it is good practice for maintaining larger enums.

Reply:
I'm on the fence about this. Order of precedence feels cleaner in this case.